docs/data/osdf/install-origin-rpm.md

```diff
@@ -0,0 +1,172 @@
+    Note that you will need the full certificate chain, not just the certificate.
+    
+    The following locations should be used (note that they are in separate directories):
+    
+    * **Host Certificate Chain**: `/etc/pki/tls/certs/pelican.crt`
+    * **Host Key**: `/etc/pki/tls/private/pelican.key`
+
+
+Installing the Origin
+---------------------
+
+The origin service is provided by the `osdf-origin` RPM.
+Install it using the following command:
+
+
+```console
+root@host # yum install --enablerepo=osg-upcoming osdf-origin
+```
+
+
+Configuring the Origin Server
+-----------------------------
+
+Configuration for a Pelican-based OSDF Origin is located in `/etc/pelican/osdf-origin.yaml`.
+
+You must configure the following:
+```
+XRootD:
+  Sitename: <RESOURCE NAME REGISTERED WITH OSG>
+```
+
+In addition, you must tell Pelican the data to export to the federation.
+An origin may export one or more directory trees, or one or more S3 buckets -- follow one of the sections below.
+(An single origin cannot export both a bucket and a directory tree.)
+
+
+
+### Configuring POSIX (directory) export
+
+Set these options to export one or more directory trees to the federation.
+
+```
+Origin:
+  StorageType: "posix"
+  Exports:
+    # You may have one or more of the following block:
+    - FederationPrefix: "<EXTERNAL OSDF NAMESPACE>"
+      StoragePrefix:    "<LOCAL FILESYSTEM DIRECTORY>"
+      Capabilities:    # Add or remove as desired
+        - Reads        # Enable authenticated reading of objects from under the directory tree through a cache
+        - PublicReads  # Enable unauthenticated reading of objects from under the directory tree through a cache
+        - DirectReads  # Enable reading objects from under the directory tree
+                       # without going through a cache
+        - Listings     # Enable directory listings
+        - Writes       # Enable writing to files in the directory tree
+```
+
+### Configuring S3 export
+
+Set these options to export one or more S3 buckets to the federation
+
+```
+Origin:
+  StorageType: "s3"
+  S3Region: "<S3 REGION IF APPLICABLE>"
+  S3ServiceUrl: "<URL OF S3 SERVER>"
+  S3UrlStyle: "path"
+  Exports:
+    # You may have one or more of the following block:
+    - FederationPrefix: "<EXTERNAL OSDF NAMESPACE>"
+      S3Bucket:         "<NAME OF S3 BUCKET>"
+      S3AccessKeyfile:  "<PATH TO S3 BUCKET ACCESS KEY>"
+      S3SecretKeyfile:  "<PATH TO S3 BUCKET SECRET KEY>"
+      Capabilities:     # Add or remove as desired
+        - Reads         # Enable authenticated reading of objects from the bucket through a cache
+        - PublicReads   # Enable unauthenticated reading of objects from the bucket through a cache
+        - DirectReads   # Enable reading objects from the bucket
+                        # without going through a cache
+        - Listings      # Enable listing bucket items
+        - Writes        # Enable writing to objects in the bucket
+```
```

Comment:
I think it makes the most sense to replace these sections with links to

- https://docs.pelicanplatform.org/federating-your-data/s3-backend
- And maybe https://docs.pelicanplatform.org/federating-your-data#origin-and-namespace-capabilities? Keeping the POSIX example around probably makes sense, actually

Reply:
You fixed this in your branch